src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> we would break the chain of the RID fallback graph.

And in this case, I presume we will update the base fallback RID (which we are discussing here) be updated to osx.10.13?